stdlib/experimental/experimental.flux

```diff
@@ -67,59 +501,641 @@ window = (
 
-// An experimental version of first
+// first returns the first record with a non-null value in the `_value` column
+// for each input table.
+// 
+// `experimental.first()` drops empty tables.
+// 
+// ## Parameters
+// - tables: Input data. Default is piped-forward data (`<-`).
+// 
+// ## Examples
+// ### Return the first non-null value in each input table
+// ```
+// import "experimental"
+// import "sampledata"
+// 
+// < sampledata.int(includeNull: true)
+// >     |> experimental.first()
+// ```
+// 
+// introduced: 0.112.0
+// tags: transformations,selectors
+// 
 builtin first : (<-tables: [{T with _value: A}]) => [{T with _value: A}]
 
-// An experimental version of last
+// last returns the last record with a non-null value in the `_value` column
+// for each input table.
+// 
+// `experimental.last()` drops empty tables.
+// 
+// ## Parameters
+// - tables: Input data. Default is piped-forward data (`<-`).
+// 
+// ## Examples
+// ### Return the last non-null value in each input table
+// ```
+// import "experimental"
+// import "sampledata"
+// 
+// < sampledata.int(includeNull: true)
+// >     |> experimental.last()
+// ```
+// 
+// introduced: 0.112.0
+// tags: transformations,selectors
+// 
 builtin last : (<-tables: [{T with _value: A}]) => [{T with _value: A}]
 
-// An experimental version of max
+// max returns the record with the highest value in the `_value` column for each
+// input table.
+// 
+// // `experimental.max()` drops empty tables.
+// 
+// ## Parameters
+// - tables: Input data. Default is piped-forward data (`<-`).
+// 
+// ## Examples
+// ### Return the row with the maximum value in each input table
+// ```
+// import "experimental"
+// import "sampledata"
+// 
+// < sampledata.int()
+// >     |> experimental.max()
+// ```
+// 
+// introduced: 0.112.0
+// tags: transformations,selectors
+// 
 builtin max : (<-tables: [{T with _value: A}]) => [{T with _value: A}]
 
-// An experimental version of min
+// min returns the record with the lowest value in the `_value` column for each
+// input table.
+// 
+// `experimental.min()` drops empty tables.
+// 
+// ## Parameters
+// - tables: Input data. Default is piped-forward data (`<-`).
+// 
+// ## Examples
+// ### Return the row with the lowest value in each input table
+// ```
+// import "experimental"
+// import "sampledata"
+// 
+// < sampledata.int()
+// >     |> experimental.min()
+// ```
+// 
+// introduced: 0.112.0
+// tags: transformations,selectors
+// 
 builtin min : (<-tables: [{T with _value: A}]) => [{T with _value: A}]
 
-// An experimental version of unique
+// unique returns all records containing unique values in the `_value` column.
+// 
+// `null` is considered a unique value.
+// 
+// `experimental.unique` outputs a single table for each input table and does
+// the following:
+// 
+// - Outputs a single record for each unique value.
+// - Leaves group keys, columns, and values unmodified.
```

Comment:
Maybe an ordered list?

Reply:
These aren't sequential. These are just general statements that describe the behavior. I'm going to restructure this section a bit.